airbyte-integrations/connector-templates/source-python-http-api/requirements.txt.hbs

```diff
@@ -1,5 +1,2 @@
-# This file is autogenerated -- only edit if you know what you are doing. Use setup.py for declaring dependencies.
--e ../../bases/airbyte-protocol
--e ../../bases/base-python
 -e ../../bases/source-acceptance-test
```

Comment:
should we publish a version of this to PyPi as well? then we'll be able to fully get rid of requirements.txt files.

Reply:
IDK if it's worth the hassle to publish this to Pypi...but we should definitely publish to Dockerhub. Let's chat offline.